### PR TITLE
Add missing par types

### PR DESCRIPTION
**Pull request by Pikeman**
Fixes StaticRiotApi.GetChampions

### CODE REVIEW

File: RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs [hunk 1]
```diff
@@ -72,6 +72,14 @@ public enum ParTypeStatic
...
-        Wind
+        Wind,
+
+        Courage,
+
+        CrimsonRush,
+
+        Flow,
+
+        Fury,
```
Comment (BenFradet) [inline RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs]: Could you add the corresponding comments (concerned champions would be great) just like it's been done for the others?

File: RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs [hunk 2]
```diff
@@ -74,12 +74,24 @@ public enum ParTypeStatic
...
 
+        /// <summary>
+        /// Flow (Yasuo).
+        /// </summary>
         Flow,
 
+        /// <summary>
+        /// Fury (Renekton).
+        /// </summary>
         Fury,
```
Comment (BenFradet) [inline RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs]: Should we specify that Yasuo's Wind is deprecated like:

```c#
/// <summary>
/// Wind (Yasuo before rework).
/// </summary>
``` 

Same for Renekton's rage?

File: RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs [hunk 3]
```diff
@@ -72,6 +72,26 @@ public enum ParTypeStatic
...
+
+        /// <summary>
+        /// Flow (Yasuo).
+        /// </summary>
+        Flow,
+
+        /// <summary>
+        /// Fury (Renekton).
+        /// </summary>
+        Fury,
```
Comment (BenFradet) [inline RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs]: Should we specify that Yasuo's Wind is deprecated like:

```c#
/// <summary>
/// Wind (Yasuo before rework).
/// </summary>
```

Same for Renekton's rage?

Comment (Pikeman) [inline RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs]: Yeah might as well!

**Comment by BenFradet** [comment]
Merging, thanks a lot!

Would you like it released quickly or are you fine building from develop for a couple of weeks?

**Comment by Pikeman** [comment]
I'm fine with develop, thanks!

**Comment by BenFradet** [comment]
2.5.3 published 👍

**Pull request merged**